ql/src/test/org/apache/hadoop/hive/ql/parse/TestUpdateDeleteSemanticAnalyzer.java

```diff
@@ -232,6 +235,8 @@ public void setup() throws Exception {
     conf.setVar(HiveConf.ConfVars.HIVEMAPREDMODE, "nonstrict");
     conf.setVar(HiveConf.ConfVars.HIVE_TXN_MANAGER, "org.apache.hadoop.hive.ql.lockmgr.DbTxnManager");
     conf.setBoolVar(HiveConf.ConfVars.HIVE_IN_TEST, true);
+    conf.set(ValidTxnList.VALID_TXNS_KEY,
```

Comment:
Why do we need this? If the objective is to cleanup the state may be conf.unset(ValidTxnList.VALID_TXN_KEY) is a better choice?

Reply:
This test doesn't go through Driver, but invokes SemanticAnalyzer directly for a transactional table, so I have to explicitly set the key. Otherwise it fails.